src/Metadata.elm

```diff
@@ -106,8 +119,10 @@ storyMetadataFromSlug slug contentDict =
         Just content ->
             Just
                 { title = content.title
-                , description = descriptionFromBody content.fullTextMarkdown
+                , description = content.summary
                 , imageSrc = imageSrcFromList content.images
+                , summary = content.summary
+                , maybeAuthor = content.maybeGroupOrIndividual
```

Comment:
Also unnecessary copy of data we have elsewhere. You can pass the content to the description field and get it's value that way.